src/modules/IscsiLioServer.rb

```diff
@@ -169,6 +169,18 @@ def getServiceStatus
         @statusOnStart = true
         @serviceStatus = true
       end
+      status = File.exist?("/etc/target/lio_setup.sh")
+      if !status
+      SCR.Execute(path(".target.bash_output"), "touch /etc/target/lio_setup.sh")
+      SCR.Execute(path(".target.bash_output"), "chmod 755 /etc/target/lio_setup.sh")
+      SCR.Execute(path(".target.bash_output"), "echo '#'!/bin/bash > /etc/target/lio_setup.sh")
+      end
+      status = File.exist?("/etc/target/tcm_setup.sh")
```

Comment:
This is not DRY, I'd put it into separate method, such as ensure_config_file_exists (?) and then call it twice

    ensure_config_file_exists("/etc/target/lio_setup.sh")
    ensure_config_file_exists("/etc/target/tcm_setup.sh")